subsys/mpsl/mpsl_init.c

```diff
@@ -140,6 +149,10 @@ static int mpsl_lib_init(const struct device *dev)
 	clock_cfg.rc_ctiv = 0;
 	clock_cfg.rc_temp_ctiv = 0;
 #endif
+	err = mpsl_support_scheduler();
```

Comment:
Can we move the call to `mpsl_support_scheduler()` above the initialization of clock parameter used below?

Reply:
Can be moved. Done